packages/office-ui-fabric-react/src/components/List/List.tsx

```diff
@@ -707,11 +768,20 @@ export class List extends BaseComponent<IListProps, IListState> implements IList
     return (renderCount === undefined ? (items ? items.length - startIndex! : 0) : renderCount);
   }
 
+  private _doFastRendering(): boolean {
```

Comment:
fastRendering is very unclear. What does that mean? 

It would be way more clear to indicate whether we're synchornously rendering all pages, or we're virtualizing. That's what we're controlling.

So we could consider renaming these to something like:

in props you could have a callback `onShouldVirtualize: (props) => boolean` with a default implementation of length > 30?

Alternative would just be rename things:

`minimumVirtualizationCount` instead of `fastRenderingItemCount`
`_shouldVirtualize` instead of `doFastRendering`